model_analyzer/result/measurement.py

```diff
@@ -36,11 +40,32 @@ def __init__(self, gpu_data, non_gpu_data, perf_config, comparator):
             Handle for ResultComparator that knows how to order measurements
         """
 
+        # average values over all GPUs
         self._gpu_data = gpu_data
+        self._avg_gpu_data = average_list(list(self._gpu_data.values()))
```

Comment:
Can `average_list` be renamed to `aggregator` and add a new parameter to the constructor which by default is equal to `average_list`?

Reply:
yes, i'll make this a priority for any future aggregation.